firebase-firestore-mixin.html

```diff
@@ -211,9 +210,9 @@
           const observer =
             `_firestoreUpdateBinding('${name}', ${args.join(',')})`
           this._createMethodObserver(observer);
+        } else {
+          this._firestoreUpdateBinding(name, ...args.map(x => this[x]));
```

Comment:
There are two issues here:
- this won’t work if the values of properties are set before we create a method observer
- even if it would work, we have a guarantee that `args` will have no items, so mapping over this is unnecessary

Reply:
> this won’t work if the values of properties are set before we create a method observer
I can see how this might be an issue, but I'm not sure that the timing requisite of such a reality is possible with the code order of a Polymer component. Can you point me to how you envisioned that being possible?

> even if it would work, we have a guarantee that args will have no items, so mapping over this is unnecessary
Good point, I can clean that up when you think this is going towards a solution.